app/views/groups/edit.html.slim

```diff
@@ -0,0 +1,4 @@
+.row
+  .col-lg-12 
+    h1= "Edit #{@group.name}"
```

Comment:
The correct syntax seems to be the following actually. Got some syntax error until I removed the `=` sign.

```
h1 Edit #{@group.name}
```